packages/cactus-core/src/main/typescript/consortium-repository.ts

```diff
@@ -1,6 +1,6 @@
 import {
-  Logger,
   Checks,
+  Logger,
```

Comment:
What is this change doing? Considering there is no other modification in this PR